release.properties

```diff
@@ -60,6 +60,7 @@
     release.info.version.major=7
     release.info.version.minor=4
     release.info.version.bug.fix=3
+    release.info.version.jakarta=true
```

Comment:
Hi @tinatian, do we really need this new property?

We're already using the properties `release.tool.dir` and `release.tool.sha` to control the code version of the repo `liferay-release-tool-ee` to be used on the release side.